### PR TITLE
Clean up module a little bit

### PR DESCRIPTION
**Pull request by limed**

### CODE REVIEW

File: eks/main.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@
 terraform {
   backend "s3" {
     bucket = "terramoz-eks-state"
-    key = "us-east-1/afrank-0/terraform.tfstate"
+    key    = "us-east-1/afrank-0/terraform.tfstate"
```
Comment (limed) [inline eks/main.tf:9]: I will say that this needs to change

File: eks/variables.tf [hunk 2]
```diff
@@ -3,18 +3,20 @@ variable "environment" {
 }
 
 variable "region" {
-  default = "us-west-1"
+  default = "us-west-2"
```
Comment (limed) [inline eks/variables.tf:6]: Is there a reason its not us-west-2 to begin with? :) #us-west-2-4lyfe

File: eks/userdata/additional-userdata.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+audisp_version="2.2.2-1"
+lifecycled_version="v3.0.2"
+
+aws s3 cp --recursive s3://audisp-json/ /tmp
+sudo rpm -i /tmp/audisp-json-$${audisp_version}.amazonlinux_x86_64.rpm
```
Comment (limed) [inline eks/userdata/additional-userdata.sh:7]: Yes the `$$` is on purpose

File: eks/main.tf (hunk 1 above)
Comment (mozafrank) [reply eks/main.tf:9]: yeah I will offer a giant WTF to terraform for not letting you use variables in the terraform {} block.

Comment (limed) [reply eks/main.tf:9]: Its less than ideal but might have to do for now

**Review by mozafrank — APPROVED** [review]
Looks great!

**Pull request merged**